samples/vmc/networks_nsxt/cgw_firewall_crud.py

```diff
@@ -39,17 +39,19 @@ def __init__(self):
         parser = argparse.ArgumentParser(
             formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
-        parser.add_argument('--refresh_token',
+        required_args = parser.add_argument_group('required arguments')
+
+        required_args.add_argument('--refresh_token',
                             required=True,
```

Comment:
Try to maintain the vertical alignment..

Reply:
Modified in the next commit